lua/resession/util.lua

```diff
@@ -168,4 +168,18 @@ M.shorten_path = function(path)
   end
 end
 
+--- Trigger a resession `User` event
+---@param event string The event name to be appended to "Resession"
+---@param delay? boolean Whether or not to delay the event asynchronously (Default: true)
+function M.event(event, delay)
+  local emit_event = function()
+    vim.api.nvim_exec_autocmds("User", { pattern = "Resession" .. event, modeline = false })
```

Comment:
I'd prefer it we could get rid of adding the `"Resession"` prefix here and just use the full name of the autocmd at the callsite. It makes it easier to find with grep

Reply:
I think this got mistakenly resolved?

Reply:
Oh, I just assumed it wouldn't make sense anymore for grepping since we now map hooks directly to event names.